dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -271,10 +292,10 @@ winrt::Control CommandBarFlyout::CreatePresenter()
 
     if (SharedHelpers::Is21H1OrHigher())
     {
-        // Since DropShadows don't play well with the entrance animation for the presenter,
+        // Since DropShadows don't play well with clip entrance animations for the presenter,
```

Comment:
Perhaps this comment should now move to the `SetPresenterName` function?

Reply:
good point